src/woo_publications/api/models/category.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ordered_model.models import OrderedModel
+
+from ..constants import InformatieCategorieChoices
+
+
+class InformatieCategorie(OrderedModel):
+    identifier = models.URLField(_("Identifier"), max_length=255, unique=True)
+    name = models.CharField(_("Name"), max_length=80)
+    name_plural = models.CharField(_("Name plural"), max_length=80)
+    definition = models.TextField(_("Definition"), blank=True)
+    origin = models.CharField(
+        _("Origin"),
+        choices=InformatieCategorieChoices.choices,
+        blank=False,
+        max_length=15,
+    )
```

Comment:
```suggestion
    origin = models.CharField(
        _("Origin"),
        choices=InformatieCategorieChoices.choices,
        blank=False,
        max_length=15,
        default=InformatieCategorieOrigins.zelf_toegevoegd,
    )
```
as discussed yesterday?

Can you also see if we can think of a proper English term for `zelf_toegevoegd` in the code? The value like that is fine so that we have dutch strings in the API spec. Maybe we can go with `user_content` or `custom_entry`?